examples/data-objects/table-view/package.json

```diff
@@ -67,10 +65,15 @@
 		"@types/react": "^17.0.44",
 		"concurrently": "^6.2.0",
 		"copyfiles": "^2.4.1",
+		"css-loader": "^1.0.0",
```

Comment:
Since you're moving deps around, would you consider updating to versions that are webpack 5 compatible officially? For example, css-loader 1.0.0 is not compatible with webpack 5 according to its peer deps, but there's a more recent version that is.

Reply:
Could consider but would do as a separate PR.  We use 1.0.0 throughout the repo currently.